src/gtk/textctrl.cpp

```diff
@@ -979,6 +989,83 @@ void wxTextCtrl::GTKSetJustification()
     }
 }
 
+#if wxUSE_SPELLCHECK
+
+wxTextSpellcheckStatus wxTextCtrl::EnableSpellcheck(const bool enable)
+{
+    if (!IsMultiLine())
+        return wxTEXT_SPELLCHECK_NOT_AVAILABLE;
+    
+    GtkTextView *textview = GTK_TEXT_VIEW(m_text);
+    wxASSERT_MSG( textview, wxS("wxTextCtrl is not a GtkTextView" ));
+    
+    wxString lang = wxLocale::GetLanguageCanonicalName(wxLANGUAGE_DEFAULT);
+    
+#if GTK_CHECK_VERSION(3,0,0)
```

Comment:
As things have moved on since 2018, I think it is no longer worthwhile supporting __WXGTK2__ with this new feature. Therefore, I will change this to just `#if wxUSE_SPELLCHECK` instead.